letsencrypt/cli.py

```diff
@@ -890,7 +892,13 @@ def _create_subparsers(helpful):
     helpful.add_group("revoke", description="Options for revocation of certs")
     helpful.add_group("rollback", description="Options for reverting config changes")
     helpful.add_group("plugins", description="Plugin options")
-
+    helpful.add(
+        "certonly", "--user-agent", type=str, default=None,
```

Comment:
Why should it be shown for `certonly`? It's not a big deal, but I feel like the best place for it is the ungrouped optional arguments such as `--email`.